courses/mi-pyt/index.html

```diff
@@ -0,0 +1,48 @@
+{% extends 'templates/_base.html' %}
+
+{% block title %}PY-MIT kurz{% endblock %}
```

Comment:
It's MI-PYT; we're not teaching Python in Massachusets yet :)
(one more "PY-MIT" is below)